src/main/java/net/imglib2/algorithm/convolution/kernel/Kernel1D.java

```diff
@@ -56,6 +56,8 @@ public class Kernel1D
 	 * @param halfKernel
 	 *            the upper half (starting at the center pixel) of the symmetric
 	 *            convolution kernel.
+	 * @return a {@link Kernel1D} used for one dimensional convolutions
+	 * @implNote op name='create.kernel1DSymmetric'
```

Comment:
Should these be Ops? What is the right name?

Reply:
Would it make sense to group all the kernel creation ops into `create.kernel.` sub-namespace? E.g. `create.kernel.symmetric1D`? Or maybe `create.kernel1D.symmetric` since these all return `Kernel1D` objects. Not sure...